### PR TITLE
Students guide update

### PR DESCRIPTION
**Pull request by anasofiapinho**
- Removed the reference to GitHub Repo
- Mentioned the Slack channel
- Added some instructions about the daily log on the teams app

### CODE REVIEW

File: students/todo.md [hunk 1]
```diff
@@ -62,15 +62,19 @@ Now there's only one thing left to say: Use all the channels there are and ask a
...
 
 <h3>Daily Log</h3>     
 The daily log is a short list of things you do each day — a summary in the form of bullet points is perfectly fine. The Teams App will be the place to put these. 
 
 Besides keeping your team in the loop, it'll get you familiar with good team practice, and with the high level of transparency that is typical for Open Source development. It will also help you greatly with self-reflection.
 
-We recommend to either combine the daily log with [a morning standup meeting](http://martinfowler.com/articles/itsNotJustStandingUp.html) or fill the log at the end of the day. 
+We recommend to either combine the daily log with [a morning standup meeting](http://martinfowler.com/articles/itsNotJustStandingUp.html) or fill the log at the end of the day.
+
+To start making your status updates, you have to login on the RGSoC Teams app and go to Status Updates on the top of he page.
```
Comment (alicetragedy) [inline students/todo.md]: We can combine this line with the intro of the daily log paragraph at the top, something like: 

"The Teams App will be the place to put these; just log into the app and go to Status Updates at the top of the page".

File: students/todo.md [hunk 2]
```diff
@@ -62,15 +62,19 @@ Now there's only one thing left to say: Use all the channels there are and ask a
 
 Most of the aspects mentioned here need to be done at the beginning of the program. Follow these steps and you'll have a great setup for all the activities.
 
-<h3>Github Private Repo</h3>      
-A separate Github repo is the best place for your team to organize the issues to work on. We will provide a private repo for each team so you can safely get on with your work.
+<h3>RGSoC-Orga Slack Team</h3>
+All the members of the team will be invited to our Slack RGsoC-Orga Team. This is currently our favourite channel of communication and we set up several channels with different purposes so that you won't miss any important information. You can also talk with each other via private message.
```
Comment (alicetragedy) [inline students/todo.md]: "All the members of the team" is a bit vague, maybe replace with "All RGSoC participants" or "All program participants".

**Comment by alicetragedy** [comment]
Awesome @anasofiapinho, thank you! I've made some comments, could you make those changes to your PR? We can definitely merge it after that!

**Comment by anasofiapinho** [comment]
@alicetragedy You can merge!

**Pull request merged**